hts-precompile/FeeHelper.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./HederaTokenService.sol";
+import "./HederaResponseCodes.sol";
+import "./IHederaTokenService.sol";
+import "./KeyHelper.sol";
+
+contract FeeHelper is KeyHelper {
+
+    function createFixedFeeForHbars(
+        uint32 amount,
+        address feeCollector
+    ) internal pure returns (IHederaTokenService.FixedFee memory fixedFee) {
+        fixedFee.amount = amount;
+        fixedFee.useHbarsForPayment = true;
+        fixedFee.feeCollector = feeCollector;
+    }
+
+    function createFixedFeeForToken(
+        uint32 amount,
+        address tokenId,
+        address feeCollector
+    ) internal pure returns (IHederaTokenService.FixedFee memory fixedFee) {
+        fixedFee.amount = amount;
+        fixedFee.tokenId = tokenId;
+        fixedFee.feeCollector = feeCollector;
+    }
+
+    function createFixedFeeForCurrentToken(
+        uint32 amount,
+        address feeCollector
+    ) internal pure returns (IHederaTokenService.FixedFee memory fixedFee) {
+        fixedFee.amount = amount;
+        fixedFee.useCurrentTokenForPayment = true;
+        fixedFee.feeCollector = feeCollector;
+    }
+
+    function createFractionalFeeWithoutLimits(
```

Comment:
`createFractionalFee()` and then below `createFractionalFeeWithMinAndMax()`, maybe?